.pre-commit-config.yaml

```diff
@@ -1,4 +1,9 @@
-exclude: ^bin|mir-ci/mir_ci/wayland/protocols
+exclude: |
+  (?x)^(
+    bin/.*|
+    mir-ci/mir_ci/wayland/protocols/.*|
+    mir-ci/mir_ci/tests/robot/platforms/zapper/ZapperHid.py
```

Comment:
Why is `ZapperHid.py` excluded here?

Reply:
It's only temporarily here, going into zapper proper soon, didn't want to deal with the different formatting between here and there.